### PR TITLE
Upgrade to newer ruby

### PR DESCRIPTION
**Pull request by ciremoussadia**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,20 @@
-FROM ruby:2.7.0-alpine
+FROM ruby:alpine3.15
```
Comment (blambeau) [inline Dockerfile:1]: Maybe we should move to startback? I'm not sure. Alpine is possibly better. Any idea?

**Review by blambeau — COMMENTED** [review]
So far so good (one question in comments).
Also, we should then add a commit to bump to 2.0.

File: Dockerfile (hunk 1 above)
Comment (blambeau) [reply Dockerfile:1]: Is that ruby 3.1 already? That's possibly worth a try.

Comment (ciremoussadia) [reply Dockerfile:1]: maybe @llambeau has a better opinion?

**Pull request merged**